var/spack/repos/builtin/packages/fabulous/package.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Fabulous(CMakePackage):
+    """FABuLOuS (Fast Accurate Block Linear krylOv Solver)
+    Library implementing Block-GMres with Inexact Breakdown and Deflated Restarting"""
+
+    homepage = "https://gitlab.inria.fr/solverstack/fabulous/"
+    git ="https://gitlab.inria.fr/solverstack/fabulous.git"
+    url = "https://gitlab.inria.fr/api/v4/projects/2083/packages/generic/source/v1.1.3/fabulous-1.1.3.tar.gz"
+    maintainers("fpruvost")
+
+    version('1.1.3', 'a75a5461984360286c26b104c1d01ac6cf7c3151bfaa42d8e980eb072981f3ef',
+            url='https://gitlab.inria.fr/api/v4/projects/2083/packages/generic/source/v1.1.3/fabulous-1.1.3.tar.gz')
+    version('1.1.2', '9bb6c15dfe5819289903fdfbf513a138eb4f255b283b11e12ecdeb8c7e3e90e7',
+            url='https://gitlab.inria.fr/solverstack/fabulous/uploads/e972b50d111410b9f3c2e0e982f145f0/fabulous-1.1.2.tar.gz')
+    version('1.1.1', '031bf0ac56a4218b766ed08aab280c0a2646414d229175a4e2e2a7708a4d6a2c',
+            url='https://gitlab.inria.fr/solverstack/fabulous/uploads/5a1cd37e407841c3cf374fd6d905509e/fabulous-1.1.1.tar.gz')
+    version('1.1.0', '7a21dd85b97df9bb113f3fef09893486c6dd9dc9fdcf2f8b6dd17f6296dfac27',
+            url='https://gitlab.inria.fr/solverstack/fabulous/uploads/10052561fa04d8575d67974d0db6d232/fabulous-1.1.0.tar.gz')
+    version("master", branch="master",  submodules=True)
+
+    # cmake's specific
+    variant("shared", default=True, description="Build as shared library")
+
+    # fabulous's specific
+    variant("examples", default=False, description="build examples and tests")
+    variant("blasmt", default=False, description="use multi-threaded blas and lapack kernels")
+
+    # Dependencies
+    depends_on("pkg-config", type='build')
```

Comment:
Is this a hard dependency on `pkg-config` or would the `pkgconfig` virtual work too (so `pkgconf` could be picked up when appropriate)?